.github/workflows/build.yaml

```
@@ -64,31 +64,31 @@ jobs:
+            cd ex02
+            make
 
-  #     - name: make re Test
-  #       run: |
-  #           cd ex02
-  #           make re
+      - name: make re Test
+        run: |
+            cd ex02
+            make re
     
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)